### PR TITLE
LPS-169348 | master

### PR DESCRIPTION
**Pull request by TaiPhamLR**
Hi @locpham97, 
This PR is for ticket https://issues.liferay.com/browse/LPS-169348. Please help me review it and leave your comments.
Thanks.

### CODE REVIEW

File: modules/apps/object/object-rest-impl/src/main/java/com/liferay/object/rest/internal/resource/v1_0/ObjectEntryResourceImpl.java [hunk 1]
```diff
@@ -619,6 +633,35 @@ private void _loadObjectDefinition(Map<String, Serializable> parameters)
...
 
+	private void _validateObjectProperties(
+		ObjectEntry objectEntry, ObjectDefinition objectDefinition,
+		Locale locale) {
+
+		Stream<String> stream = objectEntry.getProperties(
+		).keySet(
+		).stream();
+
+		List<String> propertieNames = stream.collect(
```
Comment (locpham97) [inline modules/apps/object/object-rest-impl/src/main/java/com/liferay/object/rest/internal/resource/v1_0/ObjectEntryResourceImpl.java:644]: the variable should be propertyNames

File: modules/apps/object/object-rest-impl/src/main/java/com/liferay/object/rest/internal/resource/v1_0/ObjectEntryResourceImpl.java [hunk 2]
```diff
@@ -619,6 +633,35 @@ private void _loadObjectDefinition(Map<String, Serializable> parameters)
...
+			Collectors.toCollection(ArrayList::new));
```
Comment (locpham97) [inline modules/apps/object/object-rest-impl/src/main/java/com/liferay/object/rest/internal/resource/v1_0/ObjectEntryResourceImpl.java:645]: No need to cast ArrayList, should be Collectors.toList()

Comment (TaiPhamLR) [reply modules/apps/object/object-rest-impl/src/main/java/com/liferay/object/rest/internal/resource/v1_0/ObjectEntryResourceImpl.java]: Thanks for your review. I'll fix this.

**Comment by locpham97** [comment]
LGTM

**Pull request closed without merging**